src/apps/llamapay/ethereum/llamapay.balance-fetcher.ts

```diff
@@ -71,7 +69,7 @@ export class EthereumLlamapayBalanceFetcher implements BalanceFetcher {
         const position: ContractPositionBalance = {
           type: ContractType.POSITION,
           address: stream.contract.address,
-          network: network,
+          network,
```

Comment:
Was this a needed change or just preference?

Reply:
just a personal preference. (and because it's not necessary)